UWC: Overly Cumbersome Requirements (#126)
RetroAchievements/docs: docs/guidelines/content/unwelcome-concepts.md

- Secretive or intentionally vague achievement descriptions can be used to prevent major spoilers, so long as the achievements are unmissable progression.

## Overly Cumbersome Requirements

> Achievements whose restrictions and other requirements cannot be expressed within the description character limit. A player should be able to see the requirements for an achievement from within the emulator without having to visit the website or other external source.

Reviewer: nit: Slightly more concise and direct.

```suggestion
Achievements with restrictions or requirements that cannot be fully expressed within the description character limit. A player should be able to view achievement requirements directly within their emulator, without needing to visit the RetroAchievements website.
```

- Comments on the achievement that include additional restrictions or requirements are not allowed.
- Required external resources like pastebin links are not allowed.
- Overly abbreviating a description to fit, such that the description cannot be understood easily, is not allowed.
- You may use comments to add additional *clarification* of game terms
   - Example: An achievement that specifies "no optional skills" could list what counts as optional in the comments or forum.
   - Example: "with default" DIP settings in an Arcade game is acceptable.

When these are acceptable:

>  - An approved subset whose theme has cumbersome requirements would not require the achievements to duplicate the requirements in the achievement descriptions.

Reviewer: ```suggestion
 - For an approved subset with a theme involving cumbersome requirements, achievements are not required to duplicate these requirements in their descriptions.
```

>    - Example : An RPG challenge run with a large amount of restrictions would not require them to be listed in every achievement.

Reviewer: ```suggestion
   - Example: An RPG challenge run with a large amount of restrictions would not require them to be listed in every achievement.
```

   - Be sure to include this possibility when submitting a subset approval quest if you believe it will be applicable.
   - The achievements should have some indication about following such a theme when possible.

>  - A game with a huge amount of customizable modes may find it difficult to be succinct in specifying the necessary settings for common achievements and can rely on some shorthand means to express the restrictions.

Reviewer: ```suggestion
 - A game with extensive customizable modes may find it difficult to succinctly specify the necessary settings for common achievements and may rely on some shorthand means to express those restrictions.
```

>  - A game with a huge amount of customizable modes may find it difficult to be succinct in specifying the necessary settings for common achievements and can rely on some short hand means to express the restrictions.

Reviewer: ```suggestion
 - A game with a huge amount of customizable modes may find it difficult to be succinct in specifying the necessary settings for common achievements and can rely on some shorthand means to express the restrictions.
```
